### ISSUE TITLE
Model with boolean property doen't work properly

### ISSUE THREAD

**Issue by lkmrus** [jeka-kiselyov/fastify-mongoose-api#7]
The schema will not work properly
```js
const mongoose = require('mongoose');
const mongooseConnection = await mongoose.createConnection(MONGODB_URL, { useNewUrlParser: true });

const authorSchema = mongoose.Schema({
    firstName: String,
    lastName: String,
    aGoodMan: Boolean,
    biography: String,
    created: { type: Date, default: Date.now }
});
```

You can save a doc with the property `aGoodMan` equal to  `true`.
It will not be possible to change it to `false` then.

The reason is in the method `	async apiPut` of class `DefaultModelMethods`.
```js
async apiPut(data) {
	//// this points to document (schema.methods.)
	this.schema.eachPath((pathname) => {
		if (data[pathname]) {
			this[pathname] = data[pathname];
		} else if (data[pathname] === null) {
			this[pathname] = undefined;
		}
	});

	await this.save();
}
```

It's better to change the method of checking value to something like this:
```js
async apiPut(data) {
	//// this points to document (schema.methods.)
	this.schema.eachPath((pathname) => {
		if (data[pathname] !== undefined) {
			this[pathname] = data[pathname];
		} else if (data[pathname] === null) {
			this[pathname] = undefined;
		}
	});

	await this.save();
}
```

**Comment by jeka-kiselyov** [comment]
Good point, @lkmrus 

I will take care of this.

**Comment by jeka-kiselyov** [comment]
Fixed with [80124275e1b15275e6295b4cdd91b41c599a502e](https://github.com/jeka-kiselyov/fastify-mongoose-api/commit/80124275e1b15275e6295b4cdd91b41c599a502e)

v updated to 1.1.8
